### PR TITLE
[bitnami/rabbitmq] Refine probes

### LINKED ISSUE #4731: [bitnami/rabbitmq] Question - probes misconfigured ?
**Which chart**:
Chart: rabbitmq bitnami v8.5.2
Image tag: 3.8.9-debian-10-r37

**Describe the bug**

I'm have some doubt about this default configuration. 

Does it voluntary or must be inverted ? 

Since `ping` seems doesn't check if the rabbitmq process can answer clients. 
[RabbitMQ Doc](https://www.rabbitmq.com/monitoring.html#health-checks) says for `ping` : 
> a very basic check that will succeed for the nodes that are currently waiting for a peer to sync schema from

But `check_running` seems to do this.
> will exit with an error for the nodes that are currently waiting for a peer to sync schema tables from

> rabbitmq-diagnostics check_running is a check that makes sure that the runtime is running and the RabbitMQ application on it is not stopped or paused.`

If it's voluntary could you explain me for which reasons we need to keep it as it.

**To Reproduce**

1. `helm install my-release bitnami/rabbitmq`

**Probable behavior**

* `rabbitmq-diagnostics -q ping` for liveness probe
* `rabbitmq-diagnostics -q check_running && rabbitmq-diagnostics -q check_local_alarms` for readiness probe

Based on [rabbitmq checks descriptions](https://www.rabbitmq.com/monitoring.html#readiness-probes)

It could be also nice to offer to your charts users the availability to define a startupProbe (with the same mechanism of `customReadinessProbe`). For now, there is also a workaround, by using the `sidecars` property to inject the `startupProbe` with the right indentation.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"dirty", GoVersion:"go1.15.4"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T18:49:28Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.15", GitCommit:"2adc8d7091e89b6e3ca8d048140618ec89b39369", GitTreeState:"clean", BuildDate:"2020-09-02T11:31:21Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
 I'll be glad to create a PR on this issue if you confirm the misconfiguration and add support of startupProbe.

### PR DESCRIPTION
**Pull request by jBouyoud**
**Description of the change**

Correct probes implementation according to #4731 discussion.
And add availability to define startupProbe for users who wants to.

**Benefits**

- Better readiness management in rabbitmq cluster (especially on pod movements)
- Availability to use startup probe

**Possible drawbacks**

- Possible extra cpu consummation due to `check_local_arlarms` addition in readiness probe (not tested).

**Applicable issues**

  - fixes #4731

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/rabbitmq/values.yaml [hunk 1]
```diff
@@ -456,6 +456,11 @@ customLivenessProbe: {}
 ##
 customReadinessProbe: {}
 
+## Custom Startup prove
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes
+##
+customStartupProbe: ~
```
Comment (alemorcuq) [inline bitnami/rabbitmq/values.yaml:462]: Can you set the default to `{}` as with the others?

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: bitnami/rabbitmq/values.yaml [hunk 2]
```diff
@@ -456,6 +456,11 @@ customLivenessProbe: {}
 ##
 customReadinessProbe: {}
 
+## Custom Startup prove
```
Comment (orirawlings) [inline bitnami/rabbitmq/values.yaml:459]: Typo? `probe`

**Review by orirawlings — APPROVED** [review]
I'm just a bystander who was thinking of upgrading my chart version today and noticed this pending fix. But FWIW, this LGTM (aside from a likely comment typo).

File: bitnami/rabbitmq/values.yaml (hunk 2 above)
Comment (jBouyoud) [reply bitnami/rabbitmq/values.yaml]: Fix

**Review by alemorcuq — APPROVED** [review]
LGTM!

**Pull request merged**